### PR TITLE
chore: specify devices when creating a container

### LINKED ISSUE #5989: Add --device option when creating a container
### Is your enhancement related to a problem? Please describe

Podman Desktop does not have the option to add a device like --device /dev/fuse when running the container from the UI.
My requirement does not allow me to use a terminal.

### Describe the solution you'd like

 Is it possible to do so inside the Podman Desktop? Or adding this option possibly in the "Advanced" tab?

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by feloy**
Signed-off-by: Philippe Martin <phmartin@redhat.com>

### What does this PR do?

Specify devices when creating a container, equivalent of the `--device` option

A typeahead could help the user, it could be part of a follow-up PR

### What issues does this PR fix or reference?

Fixes #5989

### How to test this PR?

Go to the Container creation form, in Advanced tab, and add devices (should exist in the podman machine), and check in the Inspect tab of the created container that the `HostConfig.Devices` info is present.

- [x] Tests are covering the bug fix or the new feature

### CODE REVIEW

**Review by deboer-tim — APPROVED** [review]
Does what it says on the tin. 

Nit: I noticed that the checkbox text was slightly bigger than others, seems everything else on this form is set to text-sm, please do that here too.

**Comment by feloy** [comment]
> Does what it says on the tin.
> 
> Nit: I noticed that the checkbox text was slightly bigger than others, seems everything else on this form is set to text-sm, please do that here too.

Good catch, thanks. I've also added some space between checkboxes, to better distinguish which label is attached to which checkbox

**Comment by axel7083** [comment]
> A typeahead could help the user, it could be part of a follow-up PR

We don't have a way to get easily and quickly the list of devices ? The only way I think about is something like `podman machine ssh ls /dev` and I do not really like it. (related to https://github.com/containers/podman/issues/24042)

File: packages/api/src/container-info.ts [hunk 1]
```diff
@@ -89,6 +89,12 @@ interface MountSettings {
 
 type MountConfig = MountSettings[];
 
+export interface DeviceMapping {
```
Comment (axel7083) [inline packages/api/src/container-info.ts:92]: Need rebase

**Review by axel7083 — APPROVED** [review]
LGTM Tested with device `/dev/dxg` (gpu on windows)

**Pull request merged**